docs/pages/development/getting-started.md

```diff
@@ -48,7 +48,7 @@ Once you have registered all of the iOS devices you would like to develop on, yo
 
 </Tabs>
 
-and installing the resulting build on your device.
+and [installing the resulting build on your device](/build/internal-distribution.md).
```

Comment:
I have the feeling that this is not the correct link (https://docs.expo.dev/build/internal-distribution/), and maybe the correct one is: https://docs.expo.dev/build-reference/variants/

Because the first one doesn't add any information about how to have the dev-env working in the resulting build, while the second doc does